### DISCUSSION TITLE
Instructions about bolus calculator

### DISCUSSION THREAD

**Discussion by iamPanka** [NightscoutFoundation/xDrip#2505] [Q&A]
Hi, I'm sorry if my question seems obvious, but I'm new to xdrip. Does the app have a bolus calculator feature? And if it has one, how to manage it? Thanks a lot

**Comment by Navid200** [comment]
@psonnera

**Comment by psonnera** [comment]
Yes it does, but you need to setup all parameters before. Mind this is not approved but just indicative. https://xdrip.readthedocs.io/en/latest/use/predictions/

**Reply by iamPanka** [reply]
so, is this probably a capture of a feature from an older version, that isn't present anymore? https://imgur.com/a/jRdHGed

**Reply by psonnera** [reply]
Yes I think so, I'd be curious to know which one.

**Reply by Navid200** [reply]
> so, is this probably a capture of a feature from an older version, that isn't present anymore? https://imgur.com/a/jRdHGed

Highly unlikely.  The reason I say that is that features are hardly ever removed from xDrip.  
I suspect it is from a personal implementation and not an official release.
